igvc_platform/launch/velodyne_lidar.launch

```diff
@@ -0,0 +1,54 @@
+<!-- -*- mode: XML -*- -->
+<!-- run velodyne_pointcloud/CloudNodelet in a nodelet manager for a VLP-16 -->
+
+<!-- Default velodyne IP Address: http://192.168.1.201/ -->
+
+<launch>
+
+  <!-- declare arguments with default values -->
```

Comment:
I would explain what some of these are. Just so dumb people like me can be lazy

Reply:
will do :+1: